indico/htdocs/js/indico/modules/registration/form/tpls/field.tpl.html

```diff
@@ -38,7 +38,8 @@
                     title="{{ 'Configure field' | i18n }}"/>
             </td>
 
-            <td ng-if="field.billableDisabled" class="icon-lock billable-disabled-warning">
+            <td ng-if="field.billableDisabled" class="icon-lock billable-disabled-warning"
+                title="{{ 'This option is locked since changing it could trigger a price change.' | i18n }}{{ regMetadata.manager ? (' But as a manager you can still modify it.' | i18n) : '' }}">
```

Comment:
that leading whitespace in the second i18n sentence is pretty ugly - i'd put it outside the translatable string